app/models/Meta.ts

```diff
@@ -14,6 +17,14 @@ export interface Address {
   longitude: string;
 }
 
+export interface LocationDetails {
+  id: number;
+  address: Address;
+  name: string;
+  recurringSchedule: RecurringSchedule;
+  inherited: boolean | Schedule;
```

Comment:
Any specific reason you chose to make this a `boolean | Schedule` instead of a `Schedule | null`? One potential issue with `boolean | Schedule` is that it allows you to pass in `true`, which doesn't really make much sense as a value. If you're doing this for compatibility with how the code is written today, is it possible to at least use the type `false | Schedule`? The `false` part is a literal type, which means it'll only accept `false` and not `true`.

Reply:
Agreed. Actually this key used to be used, but no longer is, so I just removed it altogether.